samples/subsys/shell/adc/README.rst

```diff
@@ -0,0 +1,92 @@
+.. _shell-adc-sample:
+
+Analog to Digital Converter (ADC) shell example
+###############################################
+
+Overview
+********
+
+This example provides shell access to aquiring ADC samples
+
+Requirements
+************
+
+A board with ADC support
+
+Building
+********
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/shell/adc
+   :goals: build
+   :compact:
+
+Running
+*******
+
```

Comment:
How about:
```
The default Zephyr configuration will use a serial console
for shell I/O (option :option:`CONFIG_SHELL_BACKEND_SERIAL`).
To interact with the shell, you'll need to open a serial terminal
(minicom, putty, etc.) on your development system, and connect
to the board with the following settings:

- Speed: 115200
- Data: 8 bits
- Parity: None
- Stop bits: 1

```

Reply:
@dbkinder Thank you, looks good. I have added the above to the `README.rst` file.